python/myutils/StackMaker.py

```diff
@@ -405,7 +405,7 @@ def doPlot(self):
         l2.SetNColumns(2)
 
 
-        ratio, error = getRatio(d1,allMC,self.xMin,self.xMax,"",self.maxRatioUncert)
+        ratio, error = getRatio(d1,allMC,self.xMin,self.xMax,"",self.maxRatioUncert, False)
```

Comment:
False: This adaptes the y-range of the ratio plots between two extremum of the ratio histogram.
True (default value): the y-range of the ratio plots is fixed between ymin and ymax that are defined somewhere.

> On 6 Oct 2015, at 19:12, perrozzi <notifications@github.com> wrote:
> 
> In python/myutils/StackMaker.py <https://github.com/perrozzi/Xbb/pull/10#discussion_r41293027>:
> 
> > @@ -405,7 +405,7 @@ def doPlot(self):
> >          l2.SetNColumns(2)
> >  
> >  
> > -        ratio, error = getRatio(d1,allMC,self.xMin,self.xMax,"",self.maxRatioUncert)
> > +        ratio, error = getRatio(d1,allMC,self.xMin,self.xMax,"",self.maxRatioUncert, False)
> what is the effect of this addition?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/perrozzi/Xbb/pull/10/files#r41293027>.
>